geoh5py/objects/object_base.py

```diff
@@ -305,9 +164,127 @@ def add_data(
 
         return data_objects
 
+    def add_data_to_group(
+        self, data: Union[List, Data, uuid.UUID, str], name: str
+    ) -> PropertyGroup:
+        """
+        Append data children to a :obj:`~geoh5py.groups.property_group.PropertyGroup`
+        All given data must be children of the parent object.
+
+        :param data: :obj:`~geoh5py.data.data.Data` object,
+            :obj:`~geoh5py.shared.entity.Entity.uid` or
+            :obj:`~geoh5py.shared.entity.Entity.name` of data
+        :param name: Name of a :obj:`~geoh5py.groups.property_group.PropertyGroup`.
+            A new group is created if none exist with the given name.
+
+        :return property_group: The target property_group
```

Comment:
return format and missing period
```suggestion
        :return: The target property group.
```